internal/scaffold/role.go

```diff
@@ -140,6 +140,9 @@ func UpdateRoleForResource(r *Resource, absProjectPath string) error {
 		}
 		m := &map[string]interface{}{}
 		err = yaml.Unmarshal(d, m)
+		if err != nil {
+			return fmt.Errorf("failed to parsing role(%+v): %v", role, err)
```

Comment:
```suggestion
			return fmt.Errorf("error parsing role %q: %v", role.GetName(), err)
```
The role itself will be quite lengthy. It should be sufficient to return the error and role name.

Reply:
just keeping the failed because all this part of code is using failed.